node/_tools/config.json

```diff
@@ -2,7 +2,12 @@
   "nodeVersion": "18.8.0",
   "ignore": {
     "common": ["index.js", "tmpdir.js"],
-    "fixtures": ["child-process-spawn-node.js", "empty.txt"],
+    "fixtures": [
+      "child-process-spawn-node.js",
+      "elipses.txt",
+      "empty.txt",
+      "x.txt"
```

Comment:
Sorry if I'm missing something, but why do these fixtures need to be ignored?

Reply:
Some tests will fail due to the headers that are automatically added when extracting `elipses.txt` and `x.txt` by `deno task node:setup`.
That is why I have added them to the ignore section, to keep the files without the headers.

Reply:
Got it. Thanks. I think we should not modify fixtures (not something for you to address here, just a general comment).